sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/Expression.scala

```diff
@@ -265,14 +265,21 @@ abstract class Expression extends TreeNode[Expression] {
     case single => single :: Nil
   }
 
+  protected def flatArgumentStrings: Iterator[String] = flatArguments.map {
```

Comment:
Can you leave code comments here about what's a difference from `flatArguments`?